.gitignore

```diff
@@ -12,6 +12,8 @@
 *.so
 *.dll
 *.dylib
+# ignore 'so's with appended version: libqmlbind.so.1.0.0, libqmlbindso.1.0 etc.
+libqmlbind.so.*
```

Comment:
Oh sorry, we can't remove `*.so` ...
I prefer ignoring both `*.so` and `*.so.*` because I think both covers shared objects exactly.